### DISCUSSION TITLE
A terrible problem in deployment my projects<title>

### DISCUSSION THREAD

**Discussion by amr-abd-alhameed** [community/community#113641] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Every time I try to deploy any project to git hub pages this error appears 

![error in github one](https://github.com/community/community/assets/114324069/430bef2c-2478-48b1-acdf-5143a14b7b5f)

that error is repeated for all projects even if I create a project with create react app without adding any additional dependencies that error is also found 

this is the repositories the error exists 

https://github.com/amr-abd-alhameed/engali

https://github.com/amr-abd-alhameed/seventhTemp

https://github.com/amr-abd-alhameed/saher

**Comment by madmarin** [comment, marked as answer]
Hey there!

I see you're running into issues deploying your projects to GitHub Pages, and it's popping up even with brand-new projects, even without extra dependencies. That sounds frustrating!

How about trying out Vercel for deploying your apps instead? It's a breeze to use and works great with React projects, like those created with Create React App.

Here's why it might be a good fit for you:

Easy-peasy: Vercel makes deployment a cinch, with just a few clicks or terminal commands.

React-ready: It's built to handle React projects, with special integrations to make deploying React apps a breeze.

Fast & scalable: Vercel's global infrastructure ensures speedy performance and reliable scalability for your apps.

Cool features: You'll get access to neat stuff like automatic prerendering, performance analytics, and more to enhance user experience and streamline development.

Why not give Vercel a shot and see if it sorts out your deployment woes? If you need a hand getting started, feel free to ask!

Hope this helps!

**Reply by amr-abd-alhameed** [reply]
thanks, bro i will try now!!🥰🥰
